<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebPageWidget</class>
 <widget class="QWidget" name="WebPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="UrlAddressBar" name="Url_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>26</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWebEngineViewEx" name="Web_widget" native="true"/>
   </item>
   <item>
    <widget class="QProgressBar" name="Loading_progressBar">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>5</height>
      </size>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UrlAddressBar</class>
   <extends>QWidget</extends>
   <header>UrlAddressBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QWebEngineViewEx</class>
   <extends>QWidget</extends>
   <header>QWebEngineViewEx.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
